saleor/tests/fixtures.py

```diff
@@ -3959,11 +3959,11 @@ def gift_card_tag_list(db):
 
 
 @pytest.fixture
-def gift_card(customer_user):
+def gift_card(customer_user2):
     gift_card = GiftCard.objects.create(
         code="never_expiry",
-        created_by=customer_user,
-        created_by_email=customer_user.email,
+        created_by=customer_user2,
+        created_by_email=customer_user2.email,
```

Comment:
I guess it would be better to change these fields only in the test, but this is OK with me too, we'll see what other reviewers think ;)

Reply:
I tried that to change fields only in particular test case but it' not working it was still returning `customer_user1` email instead of `customer_user2`. That's why I changed globally.

Reply:
@Smit-Parmar Did you debug it in the resolver? There is some logic around that; maybe some fields were left unchanged and it was returning the wrong value. Usually, we try to keep fixtures more general and adjust them inside the tests.

Reply:
@maarcingebala I checked the resolver [https://github.com/saleor/saleor/pull/13132#discussion_r1235171805](cause). I'm looking into this and see what can be done.